### PR TITLE
test: rewriting build_service test in python3

### PR DESCRIPTION
**Pull request by staiyeba**
Re-written build_service test in python3 for future work.

### CODE REVIEW

**Comment by alfred-bratterud** [comment]
With a pool - nice!

File: test/misc/build_services/test.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    if os.path.exists("prereq.sh"):
+        print("Installing Prereq")
+        subprocess.call("./prereq.sh")
+
+    run_boot_path=os.path.join(includeos_prefix,'bin/boot')
+    clean_build = '-cb'
+    dot = '.'
+    subprocess.call(['/bin/bash', run_boot_path,clean_build, dot,'&>',tmpfile ])
+    print(" [ PASS ]")
```
Comment (staiyeba) [inline test/misc/build_services/test.py]: use pretty library here to print

File: test/misc/build_services/test.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    subprocess.call(["git", "submodule", "update", "--init", "--recursive"])
+
+    if os.path.exists("prereq.sh"):
+        print("Installing Prereq")
+        subprocess.call("./prereq.sh")
+
+    run_boot_path=os.path.join(includeos_prefix,'bin/boot')
+    clean_build = '-cb'
+    dot = '.'
+    subprocess.call(['/bin/bash', run_boot_path,clean_build, dot,'&>',tmpfile ])
```
Comment (staiyeba) [inline test/misc/build_services/test.py]: Needs some work here, to ensure that failed tests are reported back.

File: test/misc/build_services/test.py (hunk 1 above)
Comment (staiyeba) [reply test/misc/build_services/test.py]: cannot sure pretty library for python3 code, until vmrunner, prettify , testrunner is written in python3.

**Comment by staiyeba** [comment]
closing this PR, as we further look into ctest to resolve the parallel running of tests and inform about the result of every service tested.

**Pull request closed without merging**